test/elm/string/string-test.js

```diff
@@ -8,25 +8,50 @@ describe('Concat', () => {
     setup(this, data);
   });
 
-  it.skip('should be a Concat', function () {
-    this.helloWorld.should.be.an.instanceOf(str.Concat);
-    this.helloWorldVariables.should.be.an.instanceOf(str.Concat);
-  });
+  describe('+', () => {
+    it.skip('should be a Concat', function () {
+      this.helloWorld.should.be.an.instanceOf(str.Concat);
+      this.helloWorldVariables.should.be.an.instanceOf(str.Concat);
+    });
```

Comment:
I think you can un-skip these if you change `str.Concat` to `str.Concatenate`.  At one point, the class name changed (from `Concat` to `Concatenate`) and for some reason someone skipped these tests instead of just fixing them!  But I just tested changing them to `str.Concatenate` locally and they passed.